libs/core/extended_testing_deps.txt

```diff
@@ -1 +1,2 @@
 jinja2>=3,<4
+mustache>=0.1.4,<1
```

Comment:
why was this added? we vendor the implementation of mustache

Reply:
removing here https://github.com/langchain-ai/langchain/pull/27629

Reply:
i added this because extended tests were failing for me https://github.com/langchain-ai/langchain/actions/runs/11448833821/job/31853195056